### DISCUSSION TITLE
How to fetch shop listings using the new Etsy API V3?

### DISCUSSION THREAD

**Discussion by eduFDiaz** [etsy/open-api#1063] [Q&A]
To the brilliant team of Etsy developers that worked on the new V3 APIs.

I have a web app that fetches shop listings and displays them to users. When using V2 API making a request to fetch shop listings was as simple as this.

```
const apiUrl = `https://openapi.etsy.com/v2/shops/BigOakPress/listings/active?api_key=${ETSY_API_KEY}&includes=MainImage`;
async function getListings(): Promise<any> {
  const response = await fetch(apiUrl, {
    method: 'GET',
    headers: {
      'Content-Type': 'application/json'
    }
  });

  if (response.ok) {
    const data = await response.body;
    return data;
  } else {
    throw new Error('Failed to fetch listings');
  }
}
```

How can one do the same with V3 without having users to authenticate? 

The only tutorial available for V3 revolves around OAUTH and tokens https://developers.etsy.com/documentation/tutorials/quickstart but it doesn't show how to make a simple request to fetch store listings or as a matter of fact how to hit any V3 APIs.

Can you all point to a third-party documentation/example that shows how to use the V3 APIs to fetch listings?

**Comment by Gareth064** [comment]
Here you go

https://developers.etsy.com/documentation/reference#operation/findAllListingsActive
